mongo/options/distinctoptions.go

```diff
@@ -15,6 +15,10 @@ type DistinctOptions struct {
 	// default value is nil, which means the default collation of the collection will be used.
 	Collation *Collation
 
+	// A string or document that will be included in server logs, profiling logs, and currentOp queries to help trace
+	// the operation.  The default value is nil, which means that no comment will be included in the logs.
```

Comment:
```suggestion
	// the operation. The default value is nil, which means that no comment will be included in the logs.
```
Based on past reviews and from what I've seen in the code, it seems we prefer single spaces after punctuation.

Reply:
Oh do some Go libraries use double spaces? Is there a reason to not use single spaces?

Reply:
Good catch! It's my typo. Our codebase usually follows the single-space convention, although I did see some counter examples. I also read a golang style guide (maybe from uber?) just a few weeks ago, which required single-space as I can remember, but I failed to retrieve the guide online.